CONTRIBUTING.rst

```diff
@@ -76,3 +76,13 @@ To run tox and generate a coverage report (in ``htmlcov`` directory)::
 
 **Please note**: Before a pull request can be merged, all tests must pass and
 code/branch coverage in tests must be 100%.
+
+Code Formatting
+---------------
+We make use of `black`_ for code formatting.
+
+.. _black: https://black.readthedocs.io/en/stable/installation_and_usage.html
+
+Once it is installed you can make sure the code is properly formatted by running::
```

Comment:
Could add via a pre-commit hook? I find it very helpful when a codebase has black (enivitably I forget to run it). :shrug:

Reply:
Yes, would be great I think :+1: